simon-says-files

Sebs5384 opened the pull request:
In this PR we're going to add the logic, structure and styles of our simon-says game for the class-10 of r/argentina-programa.
Changes and files are going to be added as we progress on the project.

Sebs5384 committed:
updates simon-says html

added buttons to the HTML as well as a header with the game name so we can start building our simon says game, also added the links to our JS in which we are going to implement the logic and the css link where the style will be held.

Sebs5384 committed:
add styles to buttons

these changes will enhance the visual appareance of the game and improve the user experience.

Sebs5384 committed:
create js, css, html files

These files are intended to serve as the base of our simon-say game.

Sebs5384 committed:
add audios to html

audio is being added for each button in order to be listen when pressed/pushed in our UI.

Sebs5384 committed:
add disabled attribute to buttons

putting disabled attribute so the user can't push the buttons before the game starts.

Sebs5384 committed:
updates html

making a small change on the title and IDs of some attributes in order to make them descriptive

Sebs5384 committed:
updates css

adding styles to header, turn-state and start-message also adding hidden class to hide our start button upon the game start

Sebs5384 committed:
add js file

adding the game logic which complements the css and html files.

Sebs5384 committed:
changes folder name

changing the main folder name in order to deploy it with /docs in gh-pages

Sebs5384 committed:
updates css @media

adding media query to target devices to ensure that on smaller screens the body element takes up the full height of the view

daniNKV commented on docs/simon-says.css:
```diff
@@ -0,0 +1,105 @@
+}
+
+.blue {
+  position: relative;
```
This button shouldn't have relative here

Sebs5384 committed:
updates html

removes header and strong elements on H1 and H2 in order to make it more clean, div buttons was created to held the styles of each button in our container.

Sebs5384 committed:
updates css to flex-box method

did a major change in the styles of each element in order to make it less hard-coded by using the flex-box method.

Sebs5384 committed:
updates js selectors for id names

changed the names of the IDs of the new elements contained in css.file

Sebs5384 committed:
k-10
